app/less/shipyard.less

```diff
@@ -20,6 +20,7 @@ a.ship {
     width: 100%;
     margin: 0;
     text-align: left;
+    font-size: 1.1em;
```

Comment:
Reduce font-size a little (1.2 -> 1.1) to avoid an unwanted line break ("Diamondback Explorer" is a bit too long). Should we up a bit the `a.ship` width ?